core/src/main/java/cucumber/runtime/LinePredicate.java

```diff
@@ -0,0 +1,31 @@
+package cucumber.runtime;
+
+import gherkin.events.PickleEvent;
+import gherkin.pickles.PickleLocation;
+
+import java.util.List;
+import java.util.Map;
+
+public class LinePredicate implements PicklePredicate {
+    private Map<String, List<Long>> lineFilters;
+
+    public LinePredicate(Map<String, List<Long>> lineFilters) {
+        this.lineFilters = lineFilters;
+    }
+
+    @Override
+    public boolean apply(PickleEvent pickleEvent) {
+        String picklePath = pickleEvent.uri;
+        if (!lineFilters.containsKey(picklePath)) {
+            return true;
```

Comment:
I finally remembered why it should be true, see the (new) explaining comment in the [test case](https://github.com/cucumber/cucumber-jvm/blob/master/core/src/test/java/cucumber/runtime/LinePredicateTest.java#L26-L28)